contracts/contracts/ProviderRegistry.sol

```diff
@@ -176,12 +173,12 @@ contract ProviderRegistry is
     }
 
     /**
-     * @notice Sets the new fee recipient
+     * @notice Sets a new protocol fee recipient
      * @dev onlyOwner restriction
-     * @param newFeeRecipient The address to transfer the slashed funds to.
+     * @param newFeeRecipient The address of the new protocol fee recipient
      */
-    function setNewFeeRecipient(address newFeeRecipient) external onlyOwner {
-        feeRecipient = newFeeRecipient;
+    function setNewProtocolFeeRecipient(address newFeeRecipient) external onlyOwner {
+        protocolFeeTracker.recipient = newFeeRecipient;
```

Comment:
It's a good practice to generate event in case that function is called
```suggestion
        protocolFeeTracker.recipient = newProtocolFeeRecipient;
        emit ProtocolFeeRecipientUpdated(newProtocolFeeRecipient);
```